ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/GamePanel.java

```diff
@@ -284,76 +259,60 @@ public void componentHidden(ComponentEvent e) {
-        constraints.gridx = 11 + twoBoardsGridBagOffset;
-        constraints.gridy = 1;
-        constraints.insets = new Insets(10, 0, 10, 0);
-        add(mTeamLabels.get(1), constraints);
+        constraints.gridy = 0;
+        constraints.insets = new Insets(10, 0, 10, 10);
+        jailAndLabelPanel.add(mTeamLabels.get(1), constraints);
 
         // add the Black timer
-        constraints.fill = GridBagConstraints.HORIZONTAL;
-        constraints.anchor = GridBagConstraints.BASELINE;
-        constraints.gridwidth = 3;
-        constraints.gridheight = 1;
-        constraints.gridx = 11 + twoBoardsGridBagOffset;
-        constraints.gridy = 2;
-        constraints.insets = new Insets(0, 25, 10, 25);
+        constraints.gridy = 1;
+        constraints.insets = new Insets(0, 0, 10, 10);
         // TODO: This currently assumes a two-person game.
 //		if (!ChessTimer.isNoTimer(mTimers[1]))
-//			add(new ChessTimerLabel(mTimers[1]), constraints);
+//			jailAndLabelPanel.add(new ChessTimerLabel(mTimers[1]), constraints);
 
         // add the Black Jail
-        constraints.fill = GridBagConstraints.NONE;
-        constraints.anchor = GridBagConstraints.BASELINE;
-        constraints.gridwidth = 3;
-        constraints.gridheight = 3;
-        constraints.gridx = 11 + twoBoardsGridBagOffset;
-        constraints.gridy = 3;
-        add(mJails[1], constraints);
+        constraints.gridy = 2;
+        constraints.weighty = 0.7;
+        jailAndLabelPanel.add(mJails[1], constraints);
 
         // adds the undo button
-        constraints.fill = GridBagConstraints.HORIZONTAL;
-        constraints.anchor = GridBagConstraints.BASELINE;
-        constraints.gridwidth = 3;
-        constraints.gridheight = 1;
-        constraints.gridx = 11 + twoBoardsGridBagOffset;
-        constraints.gridy = 6;
-        add(undoButton, constraints);
+        constraints.weighty = 0.0;
+        constraints.gridy = 3;
+        jailAndLabelPanel.add(undoButton, constraints);
 
         // adds the White Jail
-        constraints.fill = GridBagConstraints.NONE;
-        constraints.anchor = GridBagConstraints.BASELINE;
-        constraints.gridwidth = 3;
-        constraints.gridheight = 3;
-        constraints.gridx = 11 + twoBoardsGridBagOffset;
-        constraints.gridy = 7;
-        add(mJails[0], constraints);
+        constraints.gridy = 4;
+        constraints.weighty = 0.7;
+        jailAndLabelPanel.add(mJails[0], constraints);
 
         // adds the White timer
-        constraints.fill = GridBagConstraints.HORIZONTAL;
-        constraints.anchor = GridBagConstraints.BASELINE;
-        constraints.gridwidth = 3;
-        constraints.gridheight = 1;
-        constraints.gridx = 11 + twoBoardsGridBagOffset;
-        constraints.gridy = 6;
-        constraints.insets = new Insets(0, 0, 0, 0);
+        constraints.gridy = 5;
+        // TODO: This assumes a two-player game
+//		jailAndLabelPanel.add(new ChessTimerLabel(mTimers[0]), constraints);
```

Comment:
Can't delete it until we have a solution for Timers. Otherwise I'll forget where the go.